core/src/main/java/com/sequenceiq/cloudbreak/service/environment/EnvironmentClientService.java

```diff
@@ -57,6 +59,14 @@ public DetailedEnvironmentResponse getByCrn(String crn) {
         }
     }
 
+    public Optional<String> getCloudPlatformByCrn(String crn) {
+        try {
+            return Optional.ofNullable(getByCrn(crn).getCloudPlatform());
```

Comment:
getCloudPlatform should never return null

Reply:
that is right but I thought it is more defensive then Optional.of